graphstorm-processing/pyproject.toml

```diff
@@ -0,0 +1,58 @@
+[tool.poetry]
```

Comment:
Can we clean this file?

Reply:
Could you clarify what needs cleaning here? 

We use `pyproject.toml` as the [modern alternative](https://pip.pypa.io/en/stable/reference/build-system/pyproject-toml/) to `setup.cfg`, as introduced in [PEP-518](https://peps.python.org/pep-0518/).

The new format allows different build backends and we have chosen to use `poetry` here, hence the entries that relate to it, replacing what you would otherwise see in `setup.py` or `setup.cfg`.

Reply:
I c. This is because we need to build another package for gsprocessing.